plugin_examples/list_plugin/plugin.go

```diff
@@ -22,7 +22,8 @@ type ListPlugin struct{}
 
 func (p *ListPlugin) GetMetadata() plugin.PluginMetadata {
 	return plugin.PluginMetadata{
-		Name: "ibmcloud-list",
+		Name:          "ibmcloud-list",
+		IsCobraPlugin: true,
```

Comment:
This plug-in doesn't look like it is Cobra to me.

Reply:
You're right. I spent some time looking at the example plugins we have and apparently they are all agnostic. I was thinking of adding an example cobra plugin to include the use of `IsCobraPlugin` but I don't see any added  benefit for doing that for one field. Thoughts?

Reply:
Yeah, I don't think it is worth the time right now. The field is clear in its intention and can be doc'd elsewhere.